lib/utils/getLocales/index.ts

```diff
@@ -0,0 +1,48 @@
+import { en, fr, de, ru, zh, ko, es, hi } from "make-plural/plurals";
```

Comment:
We need to add this (or lingui) as a dependency for @klimadao/lib

Reply:
Indeed